actors/migration/nv9/miner.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	builtin3 "github.com/filecoin-project/specs-actors/v3/actors/builtin"
 	miner3 "github.com/filecoin-project/specs-actors/v3/actors/builtin/miner"
+	"github.com/filecoin-project/specs-actors/v3/actors/util/adt"
```

Comment:
In migrations, alias each versioned package to avoid errors using the wrong one.